models/engine/db_storage.py

```diff
@@ -74,3 +74,22 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Returns the object based on the class and its ID,
+        or None if not found"""
+        try:
```

Comment:
Hi peer, we were reviewing your code and we noticed that maybe the try and except is not necessary in this occasion. All in all it was a great job anyways :)